src/EpicsClient/EpicsClientFactory.h

```diff
@@ -6,12 +6,13 @@
 namespace Forwarder {
 namespace EpicsClient {
 
-///\Struct EpicsClientFactoryInit
-///\brief handles the channel access network provider.
+/// Handles the channel access network provider.
 /// Starts and stops provider on construction and destruction respectively.
 struct EpicsClientFactoryInit {
   EpicsClientFactoryInit();
   ~EpicsClientFactoryInit();
+
+  /// Returns a new instance of the EPICS client factory in a unique pointer.
```

Comment:
Too specific - remove the mention of unique pointer.
If someone changes it to a shared_ptr they won't want to change the documentation too.